src/components/CookieRejectModal.js

```diff
@@ -0,0 +1,18 @@
+import _ from 'lodash/fp'
+import { h } from 'react-hyperscript-helpers'
+import Modal from 'src/components/Modal'
+import { useStore } from 'src/libs/react-utils'
+import { authStore } from 'src/libs/state'
+
+
+const CookieRejectModal = () => {
+  const { cookiesAccepted } = useStore(authStore)
+  return cookiesAccepted === false && h(Modal, {
+    title: 'Cookies are required for Terra',
+    showCancel: false,
+    showX: false,
+    onDismiss: () => authStore.update(_.set('cookiesAccepted', true))
+  }, ['By clicking \'OK\', you agree to use cookies in Terra.'])
```

Comment:
I'm not sure single quotes are the best thing to use here. Perhaps check with user ed (unless they recommended them)?

Reply:
If it's not something specified in these [UX Guidelines page](https://www.sketch.com/s/cca3a834-985b-4165-bbae-21f615aebd2d/a/PqzMVj) or if any objections/questions about it, I'd recommend posting on #dsp-ui-ux so we can all be on the same page.

Reply:
It just looked odd to me when I saw it in the dialog, but if Lou already 'OK'd' this usage, ignore me.

Reply:
I ran this screen by the user design team. Let me double check what we use in other dialogs though.